CORTXManagerQuickstartGuide.md

```diff
@@ -2,32 +2,32 @@
 
 This guide provides a step-by-step walkthrough for getting you CORTX-Manager ready.
 
-- [1.0 Prerequisites](##10-Prerequisites)
-- [1.2 Install Cortx Manager](#12-Install-Cortx-Manager)
-- [1.3 Deploy CORTX-Manager on Test VM](#13-Deploy-CORTX-Manager-on-Test-VM)
+-   [1.0 Prerequisites](##10-Prerequisites)
```

Comment:
This guide is too old. We need to update it.
@abhijit1patil  You already have created a quick start guide. Please update

Reply:
@abhijit1patil Raise a new PR for updating md files